mapp/0.0.9/Dockerfile

```diff
@@ -0,0 +1,52 @@
+###############################################################################
+#
+#   Dockerfile for the container
+#
+#   AUTHOR: Maciej_Bak
+#   AFFILIATION: Swiss_Institute_of_Bioinformatics
+#   CONTACT: maciej.bak@unibas.ch
+#   CREATED: 06-10-2019
+#   LICENSE: GPL
```

Comment:
Are you really _sure_ you want to use a viral license for your software? It will stop people from using it... If you do, you should probably specify which GPL version.

Reply:
Why would it stop people from using it?
And what Licesnse would you suggest?

Reply:
Check this as an example:
https://github.com/zavolanlab/Dockerfiles/blob/master/cutadapt/1.16/Dockerfile

Apache License 2.0 is good to use.

Reply:
Because any viral license restricts the usage. With GPL, any code or project that wants to use MAPP _also_ has to use the _same_ license. I would recommend MIT if you want the biggest audience as people basically can do with it what they want. Alternatively, to have better protection for commercial uses of the software (but generally no restrictions), people in the open source community typically use Apache 2.0.

Reply:
I am quite sure that in all the code and tools that I have there is at least one that has GPL therefore MAPP has to be GPL. For exapmle: Bedtools, take a look here:
https://ngs.csr.uky.edu/BEDTools
Also, cutadapt is MIT and python pandas is BSD. What should I do?

Reply:
Good point. I think you indeed need to use GPL then _for the pipeline_. You could still choose to license everything that _you_ did yourself (your individual scripts & algorithms) differently, I suppose. But yeah, the way I understand it, the entire pipeline needs to be GPL then, so I guess you should leave it like that (but perhaps add the license version)